addons/controllers/cni/calicoconfig_controller.go

```diff
@@ -0,0 +1,50 @@
+// Copyright YEAR VMware, Inc. All Rights Reserved.
```

Comment:
Could we relocate `addons/controllers/cni/calicoconfig_controller.go` to something like `addons/controllers/calico/calicoconfig_controller.go`. I assume Antrea and Calico will have different folders to use.

Reply:
@vijaykatam If relocating controllers, do you think we should define ```calico``` as a separate type for the api as well?  So remove the cni grop totally, like: ```apis/calico/v1alpha1/calicoconfig_types.go``` and ```apis/antrea/v1alpha1/antreaconfig_types.go```?

Reply:
yeah lets move the controller to `addons/controllers`. API still lives at `apis/cni`

Reply:
@vijaykatam I've already moved it to ```addons/controllers``` in my current code. The question is should I move it under ```addons/controllers/cni```, under ```addons/controllers/cni/calico/``` or just under ```addons/controllers/calico/```?

Reply:
Aaah I see. I guess you mean to remove the ```cni``` layering and move it directly under ```addons/controllers/```

Reply:
@blc1996 moved it under ```addons/controllers/calico/```.